functions/Restore-DbaBackup.ps1

```diff
@@ -0,0 +1,215 @@
+function Restore-DbaBackup
```

Comment:
No problems, strict on code review is good! 

Feedback is the only way I get to improve.